packages/react/package.json

```diff
@@ -1,74 +1,32 @@
 {
-  "name": "app_scaffold",
-  "version": "2.0.0",
-  "description": "A scaffold for developers to build ZAF v2 apps",
-  "keywords": [
-    "zendesk",
-    "app",
-    "framework"
-  ],
-  "author": "Zendesk",
-  "license": "Apache-2.0",
-  "engines": {
-    "node": ">=18.12.1"
-  },
+  "name": "react_js-template",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
   "scripts": {
-    "test": "jest",
-    "watch": "webpack --watch --mode development",
-    "build:dev": "webpack --mode development",
-    "build": "webpack --mode production",
-    "lint": "standard",
-    "start": "zcli apps:server dist"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.16.0",
-    "@babel/eslint-parser": "^7.16.5",
-    "@babel/polyfill": "^7.12.1",
-    "@babel/preset-env": "^7.20.2",
-    "@babel/preset-react": "^7.18.6",
-    "babel-jest": "^29.3.1",
-    "babel-loader": "^9.1.0",
-    "clean-webpack-plugin": "^4.0.0",
-    "copy-webpack-plugin": "^11.0.0",
-    "css-loader": "^6.7.3",
-    "html-webpack-plugin": "^5.5.0",
-    "jest": "^29.3.1",
-    "jest-environment-jsdom": "^29.3.1",
-    "@testing-library/react": "^12.1.5",
-    "mini-css-extract-plugin": "^2.7.2",
-    "postcss": "^8.4.20",
-    "postcss-import": "^15.1.0",
-    "postcss-loader": "^7.0.2",
-    "postcss-preset-env": "^7.8.3",
-    "standard": "^17.0.0",
-    "webpack": "^5.75.0",
-    "webpack-cli": "^5.0.1"
+    "dev": "vite --port 3000",
+    "build": "vite build --mode production",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "format": "prettier --write .",
+    "test": "vitest"
   },
   "dependencies": {
-    "node-gyp": "^9.3.0",
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0",
-    "styled-components": "^5.3.6",
-    "@zendeskgarden/css-bedrock": "^9.0.0",
-    "@zendeskgarden/react-accordions": "^8.62.0",
-    "@zendeskgarden/react-avatars": "^8.62.0",
-    "@zendeskgarden/react-breadcrumbs": "^8.62.0",
-    "@zendeskgarden/react-buttons": "^8.62.0",
-    "@zendeskgarden/react-chrome": "^8.62.0",
-    "@zendeskgarden/react-colorpickers": "^8.62.0",
-    "@zendeskgarden/react-datepickers": "^8.62.0",
-    "@zendeskgarden/react-dropdowns": "^8.62.0",
-    "@zendeskgarden/react-forms": "^8.62.0",
-    "@zendeskgarden/react-grid": "^8.62.0",
-    "@zendeskgarden/react-loaders": "^8.62.0",
-    "@zendeskgarden/react-modals": "^8.62.0",
-    "@zendeskgarden/react-notifications": "^8.62.0",
-    "@zendeskgarden/react-pagination": "^8.62.0",
-    "@zendeskgarden/react-tabs": "^8.62.0",
-    "@zendeskgarden/react-tables": "^8.62.0",
-    "@zendeskgarden/react-tags": "^8.62.0",
-    "@zendeskgarden/react-theming": "^8.62.0",
-    "@zendeskgarden/react-tooltips": "^8.62.0",
-    "@zendeskgarden/react-typography": "^8.62.0"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "devDependencies": {
+    "@testing-library/dom": "^10.1.0",
```

Comment:
I think including Zendesk garden by default is a good way to encourage developers to use it, and would (hopefully) lead to more apps that match Zendesk's UI. Any reason we pulled this out?

Reply:
No reason at all, i will add the library as the previous one